src/+otp/+kuramotosivashinsky/KuramotoSivashinskyProblem.m

```diff
@@ -0,0 +1,59 @@
+classdef KuramotoSivashinskyProblem < otp.Problem
+    
+    methods
+        function obj = KuramotoSivashinskyProblem(timeSpan, y0, parameters)
+            obj@otp.Problem('Kuramoto-Sivashinsky', [], timeSpan, y0, parameters);
+        end
+    end
+    
+    methods
+        function soly = solution2real(soly)
+            
+            if isstruct(soly)
+                soly.y = real(ifft(soly.y));
+            else
+                soly = real(ifft(soly.')).';
+            end
+            
+        end
+        
+    end
+    
+    methods (Access = protected)
+        function onSettingsChanged(obj)
+            
+            L = obj.Parameters.L;
+            
+            N = obj.NumVars;
+            
+            div = L/(2*pi);
+            
+            k = (1i*[0:(N/2 - 1), 0, (-N/2 + 1):-1].'/div);
+            k2 = k.^2;
+            k4 = k.^4;
```

Comment:
`k2` and `k4` are always used together.  We can consolidate this to `k24 = -k.^2 - k.^4`